tools/codegen/README.md

```diff
@@ -196,10 +196,15 @@ The following generic jinja2 extensions are enabled within codegen:
 * [debug](https://jinja.palletsprojects.com/en/3.1.x/extensions/#debug-extension):
   Adds the "debug" statement to dump debugging information from within the
   template.
+* [case conversion](https://pypi.org/project/jinja2-strcase/):
+  Adds a few filters to convert `CamelCase` to `snake_case` and the other
+  way around.
 
 Additionally, codegen adds the following functions:
 
 * `bitwise_and(x, y)`: performs a bitwise and operation for two integers.
 * `bitwise_or(x, y)`: performs a bitwise or operation for two integers.
 * `bitwise_xor(x, y)`: performs a bitwise xor operation for two integers.
 * `bitwise_not(x)`: performs a bitwise not operation for an integer.
+* `re_split_function(regx, text)`: splits text based on the regex.
```

Comment:
itym `re_split` not `re_split_function`.

Reply:
good catch, autocomplete, fixed.